src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier extends ColorSupplier {
```

Comment:
```suggestion
public class FigureSupplier {
```
FigureSupplier has no connection with ColorSupplier